trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-summary-helper.ts

```diff
@@ -79,28 +52,12 @@ export const getDueDateText = (reportDueDate: Date) => {
   return `Reports${reportIsPastDue ? ' were ' : ' '}due to be received by ${formattedReportDueDate}.`;
 };
 
-/**
- * Get report period heading with all periods
- * @param submissionMonth - The submission month
- * @param reportPeriods - List of report periods
- * @returns Report period heading with the report periods listed
- */
-const getReportPeriodHeadingWithAllPeriods = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
-  const isCurrentSubmissionMonth = isSameMonth(new Date(submissionMonth), new Date());
-
-  const formattedReportPeriods = reportPeriods.map((reportPeriod) => getFormattedReportPeriodWithShortMonth(reportPeriod, true)).join(' and ');
-
-  return `${
-    isCurrentSubmissionMonth ? BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX.CURRENT_PERIOD : BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX.OPEN_PERIOD
-  }: ${formattedReportPeriods}`;
```

Comment:
I think perhaps this `BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX` constant can be simplified if not removed now that there is no variation in the header but can't see any changes to it in the diff (sorry if I have just missed it!)